charts/coturn/values.yaml

```
@@ -24,6 +24,23 @@ securityContext:
 
 coturnTurnListenPort: 3478
 coturnMetricsListenPort: 9641
+coturnTurnTlsListenPort: 5349
+
+tls:
+  enabled: false
+  certManager:
+    enabled: false
+    dnsNames: []
+    issuer:
+      create: false
+      name:
+      kind:
+      certmasterEmail:
+      solverConfiguration:
```

```suggestion
    issuerName: # name of the [Cluster]Issuer cert-manager should use
    issuerKind: ClusterIssuer # can be set to `Issuer` too
```

If we don't create the issuer anymore in the helm chart, this can become much simpler, as we only refer to an existing [Cluster]Issuer, rather than creating one (and having to provide values for all the bits and pieces that need to be configured)

---

Fixed in [3cae51b](https://github.com/wireapp/wire-server/pull/2620/commits/3cae51b9080c3c316e57fa93fe396418c348f4c9)